### PR TITLE
refactor: rename sidePanel props

### PR DESCRIPTION
**Pull request by petermakowski**
## Done

- refactor: rename sidePanel props and types

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Steps for QA.

## Fixes

Fixes: # .

## Launchpad issue

Related Launchpad maas issue in the form `lp#number`.

## Backports

In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.

Only bug and security fixes should be backported, new features should only land in main.

## Screenshots

It could be helpful to provide some screenshots to aid in QAing the change.

### CODE REVIEW

File: src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.tsx [hunk 1]
```diff
@@ -54,9 +54,8 @@ const LXDClusterDetails = (): JSX.Element => {
   const fetchedVmCluster = !gettingVmCluster && vmCluster;
 
   const loaded = clustersLoaded || fetchedVmCluster;
-  const [headerContent, setHeaderContent] = useState<KVMHeaderContent | null>(
-    null
-  );
+  const [sidePanelContent, setSidePanelContent] =
+    useState<KVMHeaderContent | null>(null);
```
Comment (ndv99) [inline src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.tsx:58]: Should KVMHeaderContent be KVMSidePanelContent?

File: src/app/kvm/views/LXDClusterDetails/LXDClusterDetailsHeader/LXDClusterDetailsHeader.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import type { SetSearchFilter } from "app/base/types";
 import urls from "app/base/urls";
 import KVMDetailsHeader from "app/kvm/components/KVMDetailsHeader";
 import { KVMHeaderViews } from "app/kvm/constants";
-import type { KVMHeaderContent, KVMSetHeaderContent } from "app/kvm/types";
+import type { KVMHeaderContent, KVMSetSidePanelContent } from "app/kvm/types";
```
Comment (ndv99) [inline src/app/kvm/views/LXDClusterDetails/LXDClusterDetailsHeader/LXDClusterDetailsHeader.tsx:14]: Likewise here

File: src/app/machines/utils.ts [hunk 3]
```diff
@@ -11,10 +11,10 @@ import { getNodeActionTitle } from "app/store/utils";
  */
 export const getHeaderTitle = (
   defaultTitle: string,
-  headerContent: MachineHeaderContent | null
+  sidePanelContent: MachineHeaderContent | null
```
Comment (ndv99) [inline src/app/machines/utils.ts:14]: Similar here - should MachineHeaderContent be MachineSidePanelContent?

File: src/app/tags/components/TagsHeader/TagHeaderForms/TagHeaderForms.tsx [hunk 4]
```diff
@@ -2,31 +2,31 @@ import AddTagForm from "../AddTagForm";
 import DeleteTagForm from "../DeleteTagForm";
 
 import { TagHeaderViews } from "app/tags/constants";
-import type { TagHeaderContent, TagSetHeaderContent } from "app/tags/types";
+import type { TagHeaderContent, TagSetSidePanelContent } from "app/tags/types";
```
Comment (ndv99) [inline src/app/tags/components/TagsHeader/TagHeaderForms/TagHeaderForms.tsx:5]: See above

File: src/app/tags/types.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import type { TagHeaderViews } from "./constants";
 
-import type { HeaderContent, SetHeaderContent } from "app/base/types";
+import type { HeaderContent, SetSidePanelContent } from "app/base/types";
```
Comment (ndv99) [inline src/app/tags/types.ts:3]: Again see above

**Review by ndv99 — APPROVED** [review]
Looks functionally sound, just a few questions about <thing>HeaderContent and whether or not they should be <thing>SidePanelContent

File: src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.tsx (hunk 1 above)
Comment (petermakowski) [reply src/app/kvm/views/LXDClusterDetails/LXDClusterDetails.tsx:58]: Yes indeed, good spot.

**Review by ndv99 — APPROVED** [review]

**Pull request merged**